printer_mods/samwiseg0/voyager_gtx_din_mount/README.md

```diff
@@ -0,0 +1,24 @@
+# Corsair Flash Voyager GTX Din Mount
+Din carrier for a Corsair Flash Voyager GTX. To be able to plug in the USB drive you can purchase a small USB extension cable located in the BOM below.
+
+The model included can be mounted to the original 2.4 din clip or the newer Trident din clip.
+
+### Printing
+  * Default voron settings
+  * No supports needed
+
+### BOM
+
+Size | Qty
+--- | ---
+M2x10 | 2
+[V2.4 PCB Din Clip](https://github.com/VoronDesign/Voron-2/blob/Voron2.4/STLs/VORON2.4/Electronics_Compartment/DIN_Brackets/pcb_din_clip_x3.stl) | 1
```

Comment:
we're going to be replacing the v2 din clips with the trident ones with r2, so I would remove references to them.